lib/tapioca/commands/todo.rb

```diff
@@ -74,10 +74,10 @@ def unresolved_constants
           .each_line
           .map do |line|
             next if line.include?("<")
-            next if line.include?("class_of")
-            line.strip.gsub("T.untyped::", "")
+            line.strip.gsub("T.untyped::", "").gsub(/T\.class_of\(([:\w]+)\)/, '\1')
           end
           .compact
+          .sort
```

Comment:
I added this `sort` so that the tests would produce deterministic results. Happy to remove it if you don't think it's correct to sort these.